Add: sub-url support

ziirish opened the pull request:
Hello @bedaes, 

I would like to know if I could implement this feature while mentioning you as a contributor?.

This feature has been asked by a user [here](https://git.ziirish.me/ziirish/burp-ui/issues/128).

Thanks.

P.S.: I cannot merge your patch as is because my source-tree is not in sync anymore with your fork. Besides, I'd like to keep supporting the original _ProxyFix_ along with this patch.

bedaes committed:
Adds sub-URL support for proxy configurations.

bedaes committed:
Add script to install burp-ui from local directory.

bedaes commented:
Hi @ziirish!

Feel free to use any code from my fork. You may mention me as contributor but the code I use originates from [here](http://flask.pocoo.org/snippets/35/). So better give credits to Peter Hansen :)

Ah, and thumbs up for creating burp-ui :+1:

ziirish commented:
Thanks for everything.

I'll implement this feature based on your patch and the original snippet and I'll add you to the contributors list with a special thanks to the Flask's community.

ziirish commented:
This is now implemented upstream: ziirish/burp-ui@d269f00

You can either configure the custom _prefix_ in your reverse-proxy (like in your patch and in the original snippet) or you can add the _prefix_ in the configuration file. Here is the [doc](http://burp-ui.readthedocs.io/en/latest/gunicorn.html#sub-root-path) if you are interested.

I also added you to the CONTRIBUTORS list.
